Pod/Classes/WPAssetViewController.m

```diff
@@ -61,6 +61,75 @@ - (void)viewDidLoad
     [self showAsset];
 }
 
++ (NSDateFormatter *)sharedDateFormater {
```

Comment:
Typo: `Formater` > `Formatter`.

Can we either move these static methods to the top of the file (then `viewDidAppear` will be next to the other lifecycle methods), or perhaps we should pull them out into a separate (private) class to handle date formatting?